frontend/src/queries/schema.ts

```diff
@@ -2089,3 +2091,24 @@ export interface ActorsPropertyTaxonomyQuery extends DataNode<ActorsPropertyTaxo
 export type ActorsPropertyTaxonomyQueryResponse = AnalyticsQueryResponseBase<ActorsPropertyTaxonomyResponse>
 
 export type CachedActorsPropertyTaxonomyQueryResponse = CachedQueryResponse<ActorsPropertyTaxonomyQueryResponse>
+
+export enum AssistantMessageType {
+    Visualization = 'visualization',
+    /**
+     * Do not use. Force the schema generator to generate a StrEnum.
+     */
+    Noop = 'noop',
```

Comment:
This value forces the scheme generator to produce a StrEnum rather than a Literal type. I haven't found a way to force the schema builder to create the enum when only one key exists.

Reply:
Ahh, that's fascinating